rest-api/etl/transform_etl_results.sh

```diff
@@ -49,7 +49,7 @@ echo "Transforming CSV files..."
 for file in ${CSV_DIR}/*.csv
 do
     filename=$(basename "$file")
-    cat $file | sed s/\"N//g > ${CSV_DIR}/transformed/$filename
+    cat $file | perl -pe 's/\"N,(?=[,\"0-9]/,/g' | sed 's/\"N$//g' > ${CSV_DIR}/transformed/$filename
```

Comment:
Can you clarify perl version requirements and print a message in this script if missing or wrong version?